src/TestSuite/ConsoleIntegrationTestCase.php

```diff
@@ -191,6 +191,18 @@ public function assertErrorRegExp($pattern, $message = '')
     }
 
     /**
+     * Asserts that `stderr` is empty
+     *
+     * @param string $message The message to output when the assertion fails.
+     * @return void
+     */
+    public function assertErrorEmpty($message = 'stderr was not empty')
+    {
+        $output = implode(PHP_EOL, $this->_err->messages());
+        $this->assertSame('', $output, $message);
+    }
```

Comment:
I imagine people might expect `assertOutputEmpty` too :)

Reply:
Ok I can add that too.